web-ext-config.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  // Global options:
+  verbose: true,
+  // Command options:
+  build: {
+    overwriteDest: true,
+  },
+  run: {
+    firefox: "nightly",
+    pref: ["extensions.experiments.enabled=true"],
+  },
+  ignoreFiles: [

```

Comment:
I think we should also add `core-addon/**/*` and `public/**/*.map`: the manifest includes the bundled code which lives in `public/addon-build`, the original code is not required in the packaged addon.